core/src/main/scala/caliban/CalibanError.scala

```diff
@@ -102,3 +106,53 @@ private object ErrorCirce {
   }
 
 }
+
+private object ErrorPlayJson {
+  import play.api.libs.json._
+  import play.api.libs.json.Json.toJson
+
+  private def locationToJson(locInfo: Option[LocationInfo]): Map[String, JsValue] =
+    Some(locInfo).collect {
+      case Some(li) => Json.arr(Json.obj("line" -> toJson(li.line), "column" -> toJson(li.column)))
+    }.fold(Map.empty[String, JsValue])(v => Map("locations" -> v))
+
+  val errorValueWrites: Writes[CalibanError] = Writes {
+    case CalibanError.ParsingError(msg, locationInfo, _, extensions) =>
+      val strictFields = Map(
+        "message"    -> toJson(s"Parsing Error: $msg"),
+        "extensions" -> toJson[Option[ResponseValue]](extensions)
+      )
+      // excluding to avoid nulls
```

Comment:
If you use the `Writes` approach above it should automatically remove `None` meaning you can just do:

```scala
JsObject(
  // Strict fields
  "message" -> s"Parsing Error: $msg",
  "extensions" -> toJson[Option[ResponseValue]](extensions)
  // Optional
  "locations" -> locationInfo
)

```